client/my-sites/stats/post-detail-highlights-section/index.tsx

```diff
@@ -58,16 +57,18 @@ export default function PostDetailHighlightsSection( {
 		post_thumbnail: post?.post_thumbnail?.URL || null,
 		title: decodeEntities( stripHTML( textTruncator( post?.title, POST_STATS_CARD_TITLE_LIMIT ) ) ),
 	};
-
-	const isJetpack = useSelector( ( state ) => siteId && isJetpackSite( state, siteId ) );
-	const isAtomic = useSelector( ( state ) => siteId && isSiteWpcomAtomic( state, siteId ) );
-	const isWPcomSite = ! isJetpack || isAtomic;
+	const { supportsEmailStats } = useSelector( ( state ) =>
+		getEnvStatsFeatureSupportChecks( state, siteId )
+	);
 
 	// postId > 0: Show the tabs for posts except for the Home Page (postId = 0).
 	// isWPcomSite: The Newsletter Stats is only covering `WPCOM sites` for now.
 	// TODO: remove the (post?.date && new Date(post?.date) >= new Date("2023-05-30")) check when the Newsletter Stats data is backfilled.
 	const isEmailTabsAvailable =
```

Comment:
Removed `isWPcomSite` checks here to make it available for all site types, including all Jetpack sites.